cpp/include/raft/sparse/csr.hpp

```diff
@@ -178,5 +178,141 @@ void weak_cc(Index_* labels,
     labels, row_ind, row_ind_ptr, nnz, N, 0, N, stream, [](Index_) { return true; });
 }
 
+/**
+ * @brief CSR representation of a MFG.
+ *
+ * @tparam IdxT node index type
+ *
+ * Same as `mfg_ellpack` except the neighbors are in CSR format.
+ *
+ * @note Valid IDs: [0, 2^(sizeof(IdxT) * 8 - 1) - 2] (see `INVALID_ID`)
```

Comment:
Can't we use std::numeirc_limits<IdxT>::max() as INVALID ID? (cugrpah is using this as INVALID ID if vertex_t is unsigned).

https://github.com/rapidsai/cugraph/blob/branch-22.06/cpp/include/cugraph/graph.hpp#L419

We may be able to move this `invalid_vertex_id` machinery to raft and re-use (this will also make transition among raft/cugrpah/cugraph-ops smoother).